eds/scripts/utils.js

```diff
@@ -159,6 +159,20 @@ export function getPartnerDataCookieValue(programType, key) {
   }
 }
 
+export function getPartnerDataCookieRootValue(key) {
```

Comment:
sorry to be picky while on PTO: this is basically the same code as getPartnerDataCookieValue, right?  can we use the getPartnerDataCookieValue method and  make the programType optional?